server/services/zigbee2mqtt/utils/convertValue.js

```diff
@@ -14,11 +14,32 @@ function convertValue(feature, state) {
       result = state === 'ON' ? 1 : 0;
       break;
     }
-    default:
+    // case 'water_leak': {
+    //   result = state === true ? 1 : 0;
+    //   break;
+    // }
+    // case 'contact': {
+    //   result = contact === true ? 1 : 0;
+    //   break;
+    // }
```

Comment:
Commented code is not necessary.
If a rollback is needed, we can get it from Git history.